django_auth_adfs/backend.py

```diff
@@ -128,11 +128,16 @@ def create_user(self, claims):
         """
         # Create the user
         username_claim = settings.USERNAME_CLAIM
+        second_username_claim = settings.SECOND_USERNAME_CLAIM
```

Comment:
Should be `guest_username_claim`.

Reply:
Great, on my way to modify ;)

Reply:
Awesome! I have to go now, so I won't be able to review or release today, but I'll try to get back to it tomorrow.

If you want help with testing, please ensure the setting allowing maintainers of this repo to push to your branch is checked. It should be somewhere on the right, I think. :relaxed: